internal/db/schema/migrations/oss/postgres/21/01_session.up.sql

```diff
@@ -0,0 +1,6 @@
+begin;
+
+alter table session_connection
+  add column user_client_ip inet not null default '::';
```

Comment:
Why `::`?

Reply:
It's an ipv6 specified address: https://en.wikipedia.org/wiki/IPv6_address#Unspecified_address

Reply:
I know what it is, I'm asking why you chose it :-)

Reply:
because it's the only "official spec" way to represent an unspecified IP address.